omod/src/main/web/dashboardwidgets/programstatus/programstatus.controller.js

```diff
@@ -313,6 +314,7 @@ export default class ProgramStatusController {
 
     enrollInProgram() {
         if (this.input.dateEnrolled && this.input.enrollmentLocation) {
+            this.requestInProgress = true;
```

Comment:
@icrc-toliveira when is this ever reset to false when the request has finished processing? It looks like it's only happening once initially in `$onInit()` as of now.

Reply:
@icrc-ifernandes kindly your ticket id is `RA-1874:Enrolment button on "Program Status"  can be pressed multiple times`  and not` RA-1874: Added variable to prevent multiple click on Enroll Button`. Kindly fix it.

Reply:
@jwnasambu the PR headline does not have to match the ticket's headline.
* The ticket headline describes a certain scope of work.
* The PR headline describes a change made to the code.

The issue can be described as "the enroll button can be clicked twice and it shouldn't" and the code change can be described as "adding a variable to greyline the enroll button".